components/ui/types.ts

```diff
@@ -0,0 +1,9 @@
+export type Colors =
```

Comment:
```suggestion
export type BaseColors =
```

Be more specific with the type naming.